sql-plan-management.md

```diff
@@ -112,25 +118,37 @@ This statement outputs the execution plan bindings at the GLOBAL or SESSION leve
 | original_sql  |  Original SQL statement after parameterization |
 | bind_sql | Bound SQL statement with hints |
 | default_db | Default database |
-| status | Status including Using, Deleted, and Invalid |
+| status | Status including Using, Deleted, Invalid, Rejected, and Pending verification|
 | create_time | Creating time |
 | update_time | Updating time |
 | charset | Character set |
 | collation | Ordering rule |
 
-### Automatically create binding
+## Baseline capturing
 
-To enable automatic binding creation, set `tidb_capture_plan_baselines` to `on`. The default value is `off`.
+To enable baseline capturing, set `tidb_capture_plan_baselines` to `on`. The default value is `off`.
 
 > **Note:**
 >
 > Because the automatic binding creation function relies on [Statement Summary](/statement-summary-tables.md), make sure to enable Statement Summary before using automatic binding.
 
-After automatic binding creation is enabled, the historical SQL statements in the Statement Summary are traversed every `bind-info-lease` (the default value is `3s`), and bindings are automatically created for SQL statements that appear at least twice.
+After automatic binding creation is enabled, the historical SQL statements in the Statement Summary are traversed every `bind-info-lease` (the default value is `3s`), and baseline capturing is automatically created for SQL statements that appear at least twice.
+
+> **Note:**
+>
+> Because TiDB has some embedded SQL statements to ensure the correctness of some functions, baseline capturing by default automatically shields these SQL statements.
```

Comment:
```suggestion
> Because TiDB has some embedded SQL statements to ensure the correctness of some features, baseline capturing by default automatically shields these SQL statements.
```